jplag/src/main/java/de/jplag/reportingV2/reportobject/model/TopComparison.java

```diff
@@ -0,0 +1,38 @@
+package de.jplag.reportingV2.reportobject.model;
```

Comment:
package reportingV2 should be reporting (in my opinion)

Reply:
The old reporting package is still present and I didn't want to mess up the both packages that is why I created it in another one. @tsaglam @sebinside  can you also provide opinion on this question?

Reply:
But this means this PR should ensure your new report generation is called in the main class and thus default behavior!